editor/plugins/node_3d_editor_plugin.cpp

```diff
@@ -2433,6 +2431,32 @@ void Node3DEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 	}
 }
 
+int Node3DEditorViewport::_get_shortcut_input_count(const String &p_name) {
+	Ref<Shortcut> check_shortcut = ED_GET_SHORTCUT(p_name);
+	if (!check_shortcut->get_events().is_empty()) {
+		return check_shortcut->get_events().size();
+	}
+	return 0;
```

Comment:
This can just be replaced with:
```cpp
return check_shortcut->get_events().size();
```
Should there be a null check here instead? Is `ED_GET_SHORTCUT` guaranteed to return a valid ref here?

Reply:
Ah you're right, `ED_GET_SHORTCUT` does error if the shortcut is nullptr. I'll simplify this as you suggested